src/mongocxx/test/collection_mocked.cpp

```diff
@@ -38,7 +38,7 @@
 #include <mongocxx/read_preference.hpp>
 #include <mongocxx/stdx.hpp>
 #include <mongocxx/options/update.hpp>
-#include <mongocxx/exception/operation_exception.hpp>
+#include <mongocxx/exception/logic_error.hpp>
```

Comment:
Oh, actually, thanks for pointing that out. This should be sorted in with the other includes. I will fix that before pushing.